accelerator/conn_mqtt.c

```diff
@@ -16,7 +16,7 @@ int main(int argc, char *argv[]) {
   struct mosquitto *mosq = NULL;
 
   // Initialize logger
-  if (logger_helper_init(LOGGER_DEBUG) != RC_OK) {
+  if (ta_logger_init() == false) {
```

Comment:
I suggest using `status_t` as return value. What do you think?

Reply:
Okay!
The reason why I did not use status_t is I was wondering include error.h in logger.h.
But I think it doesn't matter.